runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/StageInfo.java

```diff
@@ -118,7 +118,9 @@ public List<PerStepNamespaceMetrics> extractPerWorkerMetricValues() {
   private void translateKnownPerWorkerCounters(List<PerStepNamespaceMetrics> metrics) {
     for (PerStepNamespaceMetrics perStepnamespaceMetrics : metrics) {
       if (!BigQuerySinkMetrics.METRICS_NAMESPACE.equals(
-          perStepnamespaceMetrics.getMetricsNamespace())) {
+              perStepnamespaceMetrics.getMetricsNamespace())
+          && !org.apache.beam.sdk.io.kafka.KafkaSinkMetrics.METRICS_NAMESPACE.equals(
```

Comment:
Can we import this to cleanup the full path?

Reply:
Done.